AlfrescoApp/Model/Managers/SyncManager.m

```diff
@@ -536,7 +539,9 @@ - (void)syncNodes:(NSArray *)nodes includeExistingSyncNodes:(BOOL)includeExistin
             [self updateFolderSizes:YES andCheckIfAnyFileModifiedLocally:NO];
             unsigned long long totalDownloadSize = [self totalSizeForDocuments:nodesToDownload];
             
-            dispatch_async(dispatch_get_main_queue(), ^{
+            dispatch_async(dispatch_get_main_queue(), ^
+            {
```

Comment:
Code structure: Completion blocks are the exception to the "bracket on next line" rule in order to allow Xcode to auto-indent code correctly.